### PR TITLE
Master

### PR DESCRIPTION
**Pull request by mytopdog**

### CODE REVIEW

File: src/Methods/Members.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = function() {
       raw.client = _this;
       raw.tag = `${raw.username}#${raw.discriminator}`;
       raw.presence = _this.presences.get(raw.id);
-      raw.createdTimestamp = Snowflake.deconstruct(raw.id).timestamp;
+      // raw.createdTimestamp = Snowflake.deconstruct(raw.id).timestamp;
```
Comment (YoussefElshemi) [inline src/Methods/Members.js]: why was this commented out?

Comment (mytopdog) [reply src/Methods/Members.js]: didn't work
it just caused errors for me, it was for debugging I forgot to remove the comment

File: src/Methods/Members.js [hunk 2]
```diff
@@ -23,9 +23,9 @@ module.exports = function() {
         if (typeof role === 'string') roleid = role;
         if (typeof role === 'object') roleid = role.id;
         return new Promise((res) => {
-          request.req('PUT', `/guilds/${guild.id}/members/${raw.id}/roles/${roleid}`, {}, _this.token)
+          request.req('PUT', `/guilds/${guild}/members/${raw.id}/roles/${roleid}`, {}, _this.token)
```
Comment (YoussefElshemi) [inline src/Methods/Members.js]: request.req('PUT', `/guilds/${raw.guild.id}/members/${raw.id}/roles/${roleid}`, {}, _this.token)

File: src/Methods/Members.js [hunk 3]
```diff
@@ -42,9 +42,9 @@ module.exports = function() {
         if (typeof role === 'string') roleid = role;
         if (typeof role === 'object') roleid = role.id;
         return new Promise((res) => {
-          request.req('DELETE', `/guilds/${guild.id}/members/${raw.id}/roles/${roleid}`, {}, _this.token)
+          request.req('DELETE', `/guilds/${guild}/members/${raw.id}/roles/${roleid}`, {}, _this.token)
```
Comment (YoussefElshemi) [inline src/Methods/Members.js]: same as above

File: src/Methods/Members.js (hunk 1 above)
Comment (YoussefElshemi) [reply src/Methods/Members.js]: ok, please fix :)

**Pull request merged**